index.js

```diff
@@ -21,7 +21,8 @@ const {
     SourceFormatUnsupportedError,
     SourceUnsupportedError,
     SourceCorruptError,
-    RenditionTooLarge
+    RenditionTooLarge,
+    ServiceOverLoadError
```

Comment:
See note from https://github.com/adobe/asset-compute-commons/pull/71, which may lead to changes here.